src/main/java/com/sendgrid/SendGridAPI.java

```diff
@@ -0,0 +1,70 @@
+package com.sendgrid;
+
+import java.io.IOException;
+import java.util.Map;
+
+public interface SendGridAPI {
+
+      /**
+       * Initializes SendGrid
+       * @param apiKey is your SendGrid API Key: https://app.sendgrid.com/settings/api_keys
+       */
+	  public void initializeSendGrid(String apiKey);
```

Comment:
Please remove the tabs from this file and replace with spaces. This is causing the formatting to be off.